src/builtin/mother.ts

```diff
@@ -30,7 +30,7 @@ const builtInBotNames = {
   git: 'GitBot',
```

Comment:
Again this `src/builtin` folder doesnt have any test coverage yet so it would be ideal to add some testing and then make changes to here even if the changes don't seem like they'll break anything